.gitignore

```diff
@@ -102,3 +105,7 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# Pipenv
+Pipfile
+Pipfile.lock
```

Comment:
it's a bit nontraditional to exclude these two files from versioning. if you version them, you'll be more resilient to changes in underlying libraries. like, if you lose these files, and one or more dependencies updates, it could be tricky to get your code to work again without having to go in and fix things.